hmf/density_field/transfer_models.py

```diff
@@ -713,3 +720,78 @@ class EH(EH_BAO):
     """Alias of :class:`EH_BAO`."""
 
     pass
+
+
+class CLASS(FromFile):
+    """
```

Comment:
Add an `r` just before the quotes to be able to use the `\*\*` notation in the docstring (not that you have to do that).